contribs/gnodev/command_staging.go

```diff
@@ -30,6 +30,8 @@ var defaultStagingOptions = AppConfig{
 	unsafeAPI:           false,
 	lazyLoader:          false,
 	paths:               path.Join(DefaultDomain, "/**"), // Load every package under the main domain},
+	emptyBlocks:         false,
```

Comment:
I think staging could behave more like the official `gnoland` config and use empty block `true`.